change/@fluentui-react-rating-94e96782-4be8-468e-b1a3-0b2ac9a76c62.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "Rating component now correctly uses the itemLabel prop",
```

Comment:
```suggestion
  "comment": "fix: Pass missing Rating's itemLabel prop its state so RatingItem consumes it from context.",
```

Reply:
Thanks for recommending this update, is there a guideline for how these changelogs should be phrased?